core/src/main/java/org/apache/accumulo/core/client/admin/NamespaceOperations.java

```diff
@@ -84,7 +84,18 @@
 
   /**
    * Create an empty namespace with no initial configuration. Valid names for a namespace contain
-   * letters, numbers, and the underscore character.
+   * letters, numbers, and the underscore character. A safe way to ignore namespaces that do exist
+   * would be to do something like the following:
+   *
+   * <pre>
+   * {@code}
```

Comment:
This won't work. You need:

```java
  /**
   * <pre>
   * {@code
   * ... code here; note the @code tag is open ...
   * }
   * </pre>
   */
```

```

Reply:
Actually it does work and that's because I have the <pre> which I don't need if I use the {@code} correctly.  Thanks.

Reply:
Took the @code out and left the "pre" tags.  the @code did not really do anything useful.

Reply:
The main reason to use `{@code ... some code here ... }` is to not have to escape reserved HTML characters, like `<` and `>` as `&lt;` and `&gt;`, respectively. You weren't using any reserved characters, so you're fine either way. It can also be useful to some renderers to do syntax highlighting, but that's less important that displaying the contents in HTML correctly.